pkg/config_file.go

```diff
@@ -129,6 +133,14 @@ func getDefaultIntIfNil(defaultValue int, value *int) int {
 	return *value
 }
 
+func getDefaultDurationIfProvided(globalCfg, fileCfg time.Duration) time.Duration {
+	var backoff = globalCfg
+	if fileCfg != 0 {
+		backoff = fileCfg
+	}
+	return backoff
+}
```

Comment:
In other functions it is:
> get default if nil

here it is:
> get default if provided

and yet the logic is the same, I think we should stay consistent and use "not provided" instead.

```suggestion
func getDefaultDurationIfNotProvided(defaultValue, value time.Duration) time.Duration {
	if value != 0 {
		return value
	}
	return defaultValue
}
```